wgpu-native/src/device.rs

```diff
@@ -60,6 +60,9 @@ const CLEANUP_WAIT_MS: u64 = 5000;
 pub const MAX_COLOR_TARGETS: usize = 4;
 pub const MAX_VERTEX_BUFFERS: usize = 8;
 
+/// Dynamic uniform/storage buffer offsets must be aligned to this number.
+pub const DYNAMIC_BUFFER_OFFSET_ALIGN: hal::buffer::Offset = 256;
```

Comment:
If this will apply to non-dynamic buffers in the future, I could drop the 'dynamic' part.